tests/conftest.py

```diff
@@ -760,17 +766,21 @@ def _unsubscribe(topic):
 
 @pytest.fixture
 async def mqtt_mock(
-    hass,
-    mqtt_client_mock,
-    mqtt_config_entry_data,
-    mqtt_mock_entry_no_yaml_config,
-):
+    hass: HomeAssistant,
+    mqtt_client_mock: MagicMock,
+    mqtt_config_entry_data: dict[str, Any] | None,
+    mqtt_mock_entry_no_yaml_config: Callable[..., Coroutine[Any, Any, MagicMock]],
```

Comment:
Should the fourth argument not be `MqttMockType` (or as we name it from now...)?